packages/api-client/src/v2/blocks/scalar-address-bar-block/components/AddressBar.vue

```diff
@@ -110,7 +115,11 @@ events.focusAddressBar.on(() => {
           isSquare
           :method="method"
           teleport
-          @change="emits('update:method', { method: $event })" />
+          @change="
+            (payload) => {
+              emit('update:method', { value: payload })
+            }
+          " />
```

Comment:
```suggestion
          @change="
            (payload) => emit('update:method', { value: payload })
            
          " />
```
always with the extra brackets